includes/class-magic-link.php

```diff
@@ -499,9 +509,6 @@ private static function authenticate( $user_id, $token ) {
 			return false;
 		}
 
-		Reader_Activation::set_reader_verified( $user );
-		Reader_Activation::set_current_reader( $user->ID );
-
```

Comment:
It's a non-blocker, but I find it confusing that `authenticate()` doesn't authenticate.

Reply:
Hmm, that's a good point. 9d7f3ab moves the `Reader_Activation::set_current_reader` call back to the `Magic_Link::authenticate` method, but leaves the `set_reader_verified` call as a callback on the `newspack_magic_link_authenticated` hook. In my mind the verification is a side effect of authenticating via magic link, as you suggest.